tests/layers_views/views/SideBySideView.spec.js

```diff
@@ -4,17 +4,22 @@ import { PolygonLayer } from '@deck.gl/layers';
 
 import { SideBySideView } from '../../../src/views';
 import { generateViewTests, defaultArguments } from './VivView.spec';
-import { VivViewerLayer } from '../../../src/layers';
+import { VivViewerLayer, ScaleBarLayer } from '../../../src/layers';
 
 generateViewTests(SideBySideView, defaultArguments);
 
 test(`SideBySideView layer type and props check`, t => {
   const view = new SideBySideView(defaultArguments);
   const loader = { type: 'loads', isPyramid: true };
   const layers = view.getLayers({
-    props: { loader },
+    props: {
+      loader: { ...loader, PhysicalSizeXUnit: 'cm', PhysicalSizeX: 0.5 }
```

Comment:
same here. probably just create the loader object initially in one place.

Reply:
Ditto.